cmd/xcode.go

```diff
@@ -90,13 +86,19 @@ the one you usually open in Xcode, then hit Enter.
 		}
 		log.Debugf("schemes: %v", schemes)
 
-		fmt.Println()
-		selectedScheme, err := goinp.SelectFromStringsWithDefault("Select the Scheme you usually use in Xcode", 1, schemes)
-		if err != nil {
-			return printXcodeScanFinishedWithError("Failed to select Scheme: %s", err)
+		if len(schemes) == 0 {
+			return printXcodeScanFinishedWithError("No schemes found")
+		} else if len(schemes) == 1 {
+			schemeToUse = schemes[0]
+		} else {
+			fmt.Println()
+			selectedScheme, err := goinp.SelectFromStringsWithDefault("Select the Scheme you usually use in Xcode", 1, schemes)
+			if err != nil {
+				return printXcodeScanFinishedWithError("Failed to select Scheme: %s", err)
+			}
+			log.Debugf("selected scheme: %v", selectedScheme)
```

Comment:
I'd move this debug print out from this if, so that `if len(schemes) == 1 ` we'll still print the scheme, for debug purposes